get-started/index.md

```diff
@@ -234,21 +242,17 @@ web client, continue on to [Guides]({% link guides/index.md %}) for
 guides through various common actions and workflows.
 
 ## [Uninstalling Cryostat Operator](#uninstalling-cryostat-operator)
-In order to ensure that objects created by the operator and recordings created
+In order to ensure that objects created by the Operator and recordings created
 by Cryostat are properly removed, the Cryostat Operator must remain installed
 when attempting to delete the Cryostat custom resource, or any Recording
 custom resources.
 
 To completely remove Cryostat and all objects and recordings created by it:
-1. Delete any Recording custom resources.
-    - If Cryostat is no longer deployed, you must redeploy it by creating a
-      Cryostat custom resource.
-    - If the Cryostat Operator has already been uninstalled, please reinstall it
-      before deleting any Recording custom resources.
-2. Delete the Cryostat custom resource.
+1. Delete the `Cryostat` or `ClusterCryostat` custom resources (`CR`s) and
+    custom resource definitions (`CRD`s).
     - If the Cryostat Operator has already been uninstalled, please reinstall it
-      before deleting the Cryostat custom resource.
-3. Uninstall the Cryostat Operator. 
+      before deleting the `Cryostat` or `ClusterCryostat` `CR`s and `CRD`s.
+2. Uninstall the Cryostat Operator.
```

Comment:
Update this one since its only specific to a scenario?

This seems like what we need: https://olm.operatorframework.io/docs/tasks/uninstall-operator/#combine-steps-2-and-3

In AllNamespace mode, the CSV should be in `openshift-operators` and all copied CSVs will also be cleaned up if that CSV is deleted: https://olm.operatorframework.io/docs/concepts/crds/operatorgroup/#copied-csvs